### PR TITLE
Inspec aws

### PR DESCRIPTION
**Pull request by rbizzell40**
Description about what this pull request does.

Please make sure to follow the [DEV guidelines](https://gen3.org/resources/developer/dev-introduction/) before asking for review.

### New Features
- Implemented K8s Cronjob to run Chef Inspec test on eks cluster

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,4 @@ kube/services/fluentd/dockerlogs/
 *.cache
 *pyc
 !doc/terraform
+*.json
```
Comment (fauzigo) [inline .gitignore]: We might not want to add this to gitignore, this repo has a few .json files and we might add/remove/edit  anytime.

File: inspec/aws/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Gather inventory of resources from AWS environment
+
+You need python 2 or 3
+pip install aws-list-all
+
+List all resources in an AWS account, all regions, all services(*). Writes JSON files for further processing.
+
+mkvirtualenv -p $(which python3) aws
```
Comment (fauzigo) [inline inspec/aws/README.md]: `$(command -v python3)` I think is better for checking the path of an executable command, but this a README, this is just a suggestion.

File: inspec/kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+To execute this container inside a kubernetes pod you have to set the variables for the profile of the environment you want to use
```
Comment (fauzigo) [inline inspec/kubernetes/README.md]: Who needs this permission file? inspect as a service or you when executing kubectl commands or maybe something else? isn't this something that could be handled by a role/policy?

File: inspec/aws/aws_list.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+aws-list-all query --region us-east-2
```
Comment (fauzigo) [inline inspec/aws/aws_list.sh]: maybe passing a region as argument would be a better approach than hardcoding one

**Comment by themarcelor** [comment]
A Fix has been delivered to the master branch (https://github.com/uc-cdis/cloud-automation/pull/1085)
Please pull the latest changes into your branch and replay the tests.

**Comment by fauzigo** [comment]
A new PR will be submitted

**Pull request closed without merging**